content/providers/quick-start/data.yaml

```diff
@@ -340,85 +340,112 @@
+
+      # Sample the circuit on the execution backend
+      result = sampler.run(circuit).result()
+
+      quasi_dist = result.quasi_dists[0]
       print(quasi_dist)
     name: Sample a Bell State
     runMethod: sampler
   - fullCode: |-
-      from qiskit_aqt_provider import AQTProvider
-      from qiskit.primitives import BackendEstimator
-      aqt = AQTProvider('MY_TOKEN')
-      backend = aqt.get_backend('aqt_innsbruck')
-      estimator = BackendEstimator(backend)
-
-      # Specify problem hamiltonian
-      from qiskit.quantum_info import SparsePauliOp
-      
-      hamiltonian = SparsePauliOp.from_list([
-         ("II", -1.052373245772859),
-         ("IZ", 0.39793742484318045),
-         ("ZI", -0.39793742484318045),
-         ("ZZ", -0.01128010425623538),
-         ("XX", 0.18093119978423156)
-      ])
-      
-      # Define VQE ansatz, initial point and cost function
       from qiskit.circuit.library import TwoLocal
+      from qiskit.quantum_info import SparsePauliOp
+      from scipy.optimize import minimize
+
+      from qiskit_aqt_provider import AQTProvider
+      from qiskit_aqt_provider.primitives import AQTEstimator
+
+      # Select an execution backend
+      provider = AQTProvider("ACCESS_TOKEN")
+      backend = provider.get_backend("offline_simulator_no_noise")
+
+      # Instantiate an estimator on the execution backend
+      estimator = AQTEstimator(backend)
+
+      # Specify the problem Hamiltonian
+      hamiltonian = SparsePauliOp.from_list(
+          [
+              ("II", -1.052373245772859),
+              ("IZ", 0.39793742484318045),
+              ("ZI", -0.39793742484318045),
+              ("ZZ", -0.01128010425623538),
+              ("XX", 0.18093119978423156),
+          ]
+      )
+
+      # Define the VQE Ansatz, initial point, and cost function
       ansatz = TwoLocal(num_qubits=2, rotation_blocks="ry", entanglement_blocks="cz")
       initial_point = initial_point = [0] * 8
-      
+
+
       def cost_function(params, ansatz, hamiltonian, estimator):
-          energy = estimator.run(ansatz, hamiltonian, parameter_values=params).result().values[0]
-          return energy
-      
-      # Run VQE using SciPy minimizer routine
-      from scipy.optimize import minimize
-      result = minimize(cost_function, initial_point, args=(ansatz, hamiltonian, estimator), method="cobyla")
-      
-      # Print minimum eigenvalue
-      print(result.fun)
+          """Cost function for the VQE.
+
+          Return the estimated expectation value of the Hamiltonian
+          on the state prepared by the Ansatz circuit.
+          """
+          return estimator.run(ansatz, hamiltonian, parameter_values=params).result().values[0]
+
+
+        # Run the VQE using the SciPy minimizer routine
+        result = minimize(
+            cost_function, initial_point, args=(ansatz, hamiltonian, estimator), method="cobyla"
+        )
+
+        # Print the found minimum eigenvalue
+        print(result.fun)
```

Comment:
The changes appear to be formatting only. Am I seeing it correctly?

Reply:
That's correct. Our strategy was to host the examples in the provider repository, such that they can easily be tested. But this also requires sticking to the code formatting / linter rules there.

I reverted the changes and kept only the minimal ones required for compatibility with the latest release.